test/Microsoft.ML.FSharp.Tests/SmokeTests.fs

```diff
@@ -60,6 +60,7 @@ open Microsoft.ML.Legacy.Data
 open Microsoft.ML.Legacy.Transforms
 open Microsoft.ML.Legacy.Trainers
 open Microsoft.ML.Runtime.Api
+open Microsoft.ML.Runtime.EntryPoints
```

Comment:
I think this is unneccessary - I added it to see if it helped the F# test failing on `'Data.TextLoader' not found`.